plugins/GSdx/Renderers/HW/GSRendererHW.cpp

```diff
@@ -37,43 +37,48 @@ GSRendererHW::GSRendererHW(GSTextureCache* tc)
 	, m_userhacks_tcoffset_x(0)
 	, m_userhacks_tcoffset_y(0)
 	, m_channel_shuffle(false)
-	, m_lod(GSVector2i(0,0))
+	, m_lod(GSVector2i(0, 0))
 {
 	m_mipmap = theApp.GetConfigI("mipmap_hw");
 	m_upscale_multiplier = theApp.GetConfigI("upscale_multiplier");
-	m_large_framebuffer  = theApp.GetConfigB("large_framebuffer");
+	m_conservative_framebuffer = theApp.GetConfigB("conservative_framebuffer");
```

Comment:
I don't think I see any other place where you inverted the option(or I haven't spotted it) so invert it here.

Reply:
It's changed from inverting to changing all the renderer code to not need inverting it in the code.